projects/packages/forms/src/dashboard/components/InboxStatusToggle/useStatusTabs.js

```diff
@@ -0,0 +1,43 @@
+import { useEntityRecords } from '@wordpress/core-data';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Returns a formatted tab label with count.
+ *
+ * @param {string} label - The label for the tab.
+ * @param {number} count - The count to display.
+ * @return {string} The formatted label.
+ */
+function getTabLabel( label, count ) {
+	/* translators: 1: Tab label, 2: Count */
+	return sprintf( __( '%1$s (%2$s)', 'jetpack-forms' ), label, count || 0 );
+}
+
+/**
+ * Custom hook to get status tabs for the inbox view.
+ *
+ * @return {Array<{label: string, value: string}>} The status tabs.
+ */
+export default function useStatusTabs() {
+	const { totalItems: totalItemsInbox } = useEntityRecords( 'postType', 'feedback', {
```

Comment:
We need to have access to the current query in order to show the correct totals like we[ did in trunk ](https://github.com/Automattic/jetpack/pull/43358/files#diff-f6dedbab4016eb9af19b412e0c0be98ae4342e84c9c69aa770554719c50983b7L47).

Since the tabs are inside DataViews now, we don't even need to access it through the `selector`. I'll check tomorrow where the `export` button component lives in this PR, but if it has also access to the current query, we could also remove the `getCurrentQuery` selector altogether.

Reply:
I've updated this. The logic for the tabs is now in the main component, and the logic for the query is there, but I'm passing the queryArgs/currentQuery in from the parent dataviews component now.